tide-core/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "tide-core"
+version = "0.2.0"
+edition = "2018"
+authors = [
+  "Tide Developers",
+]
+description = "Core types and traits for Tide web framework"
+documentation = "https://docs.rs/tide-core"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rustasync/tide"
+autoexamples = true
```

Comment:
This is the default so doesn't need to be specified.

Reply:
Yeah, this is a remnant from a previous attempt. I've removed it locally :smile: